modules/cron/cron.go

```diff
@@ -12,80 +12,107 @@ import (
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/log"
 	"code.gitea.io/gitea/modules/setting"
+	"code.gitea.io/gitea/modules/sync"
```

Comment:
* copyright header
* order of imports
* path to cron package may need to be updated (it is now github.com/gogs/cron)

Reply:
This is only a refactor of status table. I don't understand what did you mean.

Reply:
Sorry, GitHub doesn't let me comment on lines that aren't edited. I see your point, and agree that copyright header probably doesn't need to be added.

The other items I listed are github library import comes before code.gitea.io imports, and also the github import needs to be updated to use correct path (gogits org renamed to gogs)

Reply:
done.